src/kvstore/kvstore_dist_server.h

```diff
@@ -202,12 +203,12 @@ class KVStoreDistServer {
       if (parts[0] == "filename") {
         parts[1] = "rank" + std::to_string(ps::MyRank()) + "_" + parts[1];
       }
-      char * ckey = new char [parts[0].length()+1];
-      std::strcpy (ckey, parts[0].c_str());
+      char * ckey = new char[parts[0].length() + 1];
+      std::sprintf(ckey, "%s", parts[0].c_str());
```

Comment:
why? this would be slower

Reply:
Lint complained that strcpy is not mxnet's style. It asked me to use sprintf or snprintf